doc/guides/security_release_process.md

```diff
@@ -70,9 +70,9 @@ a security release.
   * In the docker-node repository run the
     [generate-stackbrew-library.sh]( https://github.com/nodejs/docker-node/blob/master/generate-stackbrew-library.sh)
     script and replace official-images/library/node with the output generated.
-```shell
-$ ./generate-stackbrew-library.sh > .../official-images/library/node
-```
+    ```shell
```

Comment:
While we're in here, can we change `shell` to `console`? At least in GitHub's rendering, `shell` changes the `>` to a highlighted color, probably because it thinks it's a shell script, but of course, it's just a command line and should not have varied highlighting in what the user is supposed to type.

Reply:
OK, I'll do that. I wasn't sure what the convention was because i've also seen "text" or just blank used in the repo